crowbar_framework/app/helpers/nodes_helper.rb

```diff
@@ -44,6 +44,8 @@ def piechart_tooltip(values)
       result.push t("unknown", count: values[1], scope: "nodes.index.status_pie") if values[1] > 0
       result.push t("unready", count: values[2], scope: "nodes.index.status_pie") if values[2] > 0
       result.push t("pending", count: values[3], scope: "nodes.index.status_pie") if values[3] > 0
+      result.push t("crowbar_upgrade", count: values[4],
+                    scope: "nodes.index.status_pie") if values[4] > 0
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.